docs/tutorials/hello.rst

```diff
@@ -64,3 +64,9 @@ change the scroll method's argument.
     in a way that MicroPython can't understand. Check you're not missing any
     special characters like ``"`` or ``:``. It's like putting. a full stop in
     the middle of a sentence. It's hard to understand exactly what you mean.
+
+    Your microbit may stop responding: you cannot flash new code to it or
```

Comment:
> after I held down the reset button for a few seconds 
Did you just hold down the reset button, or did it while it was power cycling? https://www.mbed.com/en/development/hardware/prototyping-production/daplink/daplink-on-kl26z/